src/main/java/gov/cabinetoffice/gap/spotlight/publisher/enums/ResponseTypeEnum.java

```diff
@@ -0,0 +1,29 @@
+package gov.cabinetoffice.gap.spotlight.publisher.enums;
+
+import java.util.Map;
+
+import static gov.cabinetoffice.gap.spotlight.publisher.costants.ValidationMaps.LONG_ANSWER_VALIDATION;
+import static gov.cabinetoffice.gap.spotlight.publisher.costants.ValidationMaps.NO_VALIDATION;
+import static gov.cabinetoffice.gap.spotlight.publisher.costants.ValidationMaps.NUMERIC_ANSWER_VALIDATION;
+import static gov.cabinetoffice.gap.spotlight.publisher.costants.ValidationMaps.SHORT_ANSWER_VALIDATION;
+import static gov.cabinetoffice.gap.spotlight.publisher.costants.ValidationMaps.SINGLE_FILE_UPLOAD_VALIDATION;
+
+
+public enum ResponseTypeEnum {
+
+    // TODO update validation on AddressInput and Date
```

Comment:
Is this going to be addressed in another ticket?

Reply:
removed  the comment as no idea what the context around it is.